src/sentry/models/event.py

```diff
@@ -182,16 +182,20 @@ def get_tag(self, key):
                 return v
         return None
 
+    @property
+    def release(self):
+        return self.get_tag('sentry:release')
+
     @property
     def dist(self):
         return self.get_tag('sentry:dist')
 
     def as_dict(self):
         # We use a OrderedDict to keep elements ordered for a potential JSON serializer
         data = OrderedDict()
-        data['id'] = self.event_id
```

Comment:
I don't fully understand the meaning/repercussions of changing this (and the test), can you explain?

Reply:
This code is only used in the JSON export. Neither UI nor plugins depend on this. The idea here is to make that output compatible to ingestion, so we can easily take such a payload and process it again, etc

Reply:
Ah, got it.

Reply:
Thank you @mitsuhiko @jan-auer for changing this. It definitely annoyed me before that we were using `id` instead of `event_id` in this serialization but I had just assumed that changing it would blow up everything.

Reply:
Etching ever closer to being able to re-ingest off `to_json` :)